File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task9_CLAUDE_claude-3-5-sonnet-20240620.java

import java.util.Scanner;

public class OddOrEven {
    public static String oddOrEven(int[] arr) {
        if (arr.length == 0) {
            return "even";
        }
        
        int sum = 0;
        for (int num : arr) {
            sum += num;
        }
        
        return (sum % 2 == 0) ? "even" : "odd";
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number of elements: ");
        int n = scanner.nextInt();
        
        int[] arr = new int[n];
        System.out.println("Enter the elements:");
        for (int i = 0; i < n; i++) {
            arr[i] = scanner.nextInt();
        }
        
        System.out.println(oddOrEven(arr));
        scanner.close();
    }
}

File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task9_CLAUDE_claude-3-5-sonnet-20240620.c

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

char* oddOrEven(int* arr, int size) {
    if (size == 0) {
        return "even";
    }
    
    int sum = 0;
    for (int i = 0; i < size; i++) {
        sum += arr[i];
    }
    
    return (sum % 2 == 0) ? "even" : "odd";
}

int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    
    int* arr = (int*)malloc(n * sizeof(int));
    printf("Enter the elements:\
");
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
    
    char* result = oddOrEven(arr, n);
    printf("%s\
", result);
    
    free(arr);
    return 0;
}
